app/models/compute_resources/foreman/model/gce.rb

```diff
@@ -196,6 +196,13 @@ def user_data_supported?
       true
     end
 
+    def console(uuid)
+      vm = find_vm_by_uuid(uuid)
```

Comment:
```ruby
raise Foreman::Exception, N_('The console is not available because the VM is not powered on') unless vm.ready?
```
Would it make sense to add this?